pkg/component/observability/monitoring/prometheus/garden/testdata/seed.prometheusrule.test.yaml

```diff
@@ -2,20 +2,20 @@ rule_files:
 - seed.prometheusrule.yaml
 
 tests:
-  - name: SeedExtensionsNotReady:SmoothOutFlapping
+  - name: SeedConditionFailing:SmoothOutFlapping
```

Comment:
Maybe split the commit `Use one single alert for all conditions instead` into 2 commits so that the diff introducing the new `SeedConditionFailing` alert is not "hidden" in the diff.